examples/components/Tooltip.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const TOP = 'top';
+const LEFT = 'left';
+const TOP_LEFT = 'top-left';
+
+const getTooltipStyles = (target, direction) => {
```

Comment:
Now outside of core Draftail. Should ideally be replaced with a proper tooltip component.